package.json

```diff
@@ -35,6 +35,10 @@
     }
   },
   "dependencies": {
+    "@types/gapi": "^0.0.36",
+    "@types/gapi.client": "^1.0.1",
+    "@types/gapi.client.drive": "^3.0.3",
+    "@types/google.picker": "^0.0.33",
```

Comment:
can we have @types dependencies saved as devDependencies? they're not needed for production